applications/OptimizationApplication/python_scripts/algorithms/standardized_NLOPT_objective.py

```diff
@@ -0,0 +1,162 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.OptimizationApplication as KratosOA
+from KratosMultiphysics.OptimizationApplication.utilities.optimization_problem import OptimizationProblem
+from KratosMultiphysics.OptimizationApplication.responses.response_routine import ResponseRoutine
+from KratosMultiphysics.OptimizationApplication.controls.master_control import MasterControl
+from KratosMultiphysics.OptimizationApplication.utilities.component_data_view import ComponentDataView
+from KratosMultiphysics.OptimizationApplication.utilities.logger_utilities import DictLogger
+from KratosMultiphysics.OptimizationApplication.utilities.logger_utilities import TimeLogger
+from KratosMultiphysics.OptimizationApplication.utilities.helper_utilities import CallOnAll
+import numpy
+import time as timer
+import datetime
+
+class StandardizedNLOPTObjective(ResponseRoutine):
+    """Standardized objective response function
+
+    This class transformed a user-given optimization problem into the standard format.
+    Supported objective types:
+        "minimization",
+        "maximization"
+
+    """
+    def __init__(self, parameters: Kratos.Parameters, master_control: MasterControl, optimization_problem: OptimizationProblem, required_buffer_size: int = 2):
```

Comment:
Why not to inherit from standartized objective.py and reloud only few functions. A lot of functions are identical.

Reply:
I have overwritten almost function only GetValue and GetStandardizedValue are the same, for the sake of clarity let's  keep it fully deparate